src/dir/glob.cr

```diff
@@ -1,266 +1,299 @@
 class Dir
+  # Returns an array of all files that match against any of *patterns*.
+  #
+  # The pattern syntax is similar to shell filename globbing, see `File.match?` for details.
+  #
+  # NOTE: Path separator in patterns needs to be always `/`. The returned file names use system-specific path separators.
```

Comment:
This should be fixed to match both.

Reply:
It matches both, it's just the pattern itself needs to use `/`. And this is IMHO a good solution because backslasashes are used as escape characters in both Crystal string and POSIX paths.